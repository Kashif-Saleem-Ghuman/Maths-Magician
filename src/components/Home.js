import React from 'react';

export default function Home() {
  return (
    <>
      <header>
        <h1>
          Math Magicians
        </h1>
      </header>
      <main>
        <h2>Welcome to Our page!</h2>
        <p className="paragraph">
          Lorem ipsum dolor sit amet consectetur adipisicing elit.
          Voluptas suscipit deserunt commodi iusto quasi voluptatem
          incidunt modi perspiciatis ipsam
          dolorum at ut pariatur sunt veniam saepe,
          eveniet quis consequuntur fuga assumenda magni amet cum
          Iusto enim fugiat provident aliquam nobis laborum
          aperiam similique aspernatur obcaecati sunt
          asperiores quidem totam numquam at hic fuga temporibus
          Distinctio commodi nostrum laudantium accusantium quam.
        </p>
        <p className="paragraph2">
          Lorem ipsum dolor sit amet consectetur adipisicing elit.
          Voluptas suscipit deserunt commodi iusto quasi voluptatem
          incidunt modi perspiciatis ipsam
          dolorum at ut pariatur sunt veniam saepe,
          eveniet quis consequuntur fuga assumenda magni amet cum
          Iusto enim fugiat provident aliquam nobis laborum
          aperiam similique aspernatur obcaecati sunt.
        </p>
      </main>
      <footer>
        <span className="footer-text">Created by Kashif Saleem</span>
        <a className="github" target="_blank" rel="noreferrer" href="https://github.com/Kashif-Saleem-Ghuman">Github</a>
      </footer>

    </>
  );
}
