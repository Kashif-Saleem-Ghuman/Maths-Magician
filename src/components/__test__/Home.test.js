import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('renders when the home is updated', () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
