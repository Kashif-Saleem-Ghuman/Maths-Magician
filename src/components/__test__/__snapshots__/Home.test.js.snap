// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when the 1`] = `
Array [
  <main>
    <h2>
      Welcome to Our page!
    </h2>
    <p
      className="paragraph"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas suscipit deserunt commodi iusto quasi voluptatem incidunt modi perspiciatis ipsam dolorum at ut pariatur sunt veniam saepe, eveniet quis consequuntur fuga assumenda magni amet cum Iusto enim fugiat provident aliquam nobis laborum aperiam similique aspernatur obcaecati sunt asperiores quidem totam numquam at hic fuga temporibus Distinctio commodi nostrum laudantium accusantium quam.
    </p>
    <p
      className="paragraph2"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas suscipit deserunt commodi iusto quasi voluptatem incidunt modi perspiciatis ipsam dolorum at ut pariatur sunt veniam saepe, eveniet quis consequuntur fuga assumenda magni amet cum Iusto enim fugiat provident aliquam nobis laborum aperiam similique aspernatur obcaecati sunt.
    </p>
  </main>,
  <footer>
    <span
      className="footer-text"
    >
      Created by Kashif Saleem
    </span>
    <a
      className="github"
      href="https://github.com/Kashif-Saleem-Ghuman"
      rel="noreferrer"
      target="_blank"
    >
      Github
    </a>
  </footer>,
]
`;

exports[`renders when the home is updated 1`] = `
Array [
  <main>
    <h2>
      Welcome to Our page!
    </h2>
    <p
      className="paragraph"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas suscipit deserunt commodi iusto quasi voluptatem incidunt modi perspiciatis ipsam dolorum at ut pariatur sunt veniam saepe, eveniet quis consequuntur fuga assumenda magni amet cum Iusto enim fugiat provident aliquam nobis laborum aperiam similique aspernatur obcaecati sunt asperiores quidem totam numquam at hic fuga temporibus Distinctio commodi nostrum laudantium accusantium quam.
    </p>
    <p
      className="paragraph2"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas suscipit deserunt commodi iusto quasi voluptatem incidunt modi perspiciatis ipsam dolorum at ut pariatur sunt veniam saepe, eveniet quis consequuntur fuga assumenda magni amet cum Iusto enim fugiat provident aliquam nobis laborum aperiam similique aspernatur obcaecati sunt.
    </p>
  </main>,
  <footer>
    <span
      className="footer-text"
    >
      Created by Kashif Saleem
    </span>
    <a
      className="github"
      href="https://github.com/Kashif-Saleem-Ghuman"
      rel="noreferrer"
      target="_blank"
    >
      Github
    </a>
  </footer>,
]
`;

exports[`renders when there is quote 1`] = `
Array [
  <main>
    <h2>
      Welcome to Our page!
    </h2>
    <p
      className="paragraph"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas suscipit deserunt commodi iusto quasi voluptatem incidunt modi perspiciatis ipsam dolorum at ut pariatur sunt veniam saepe, eveniet quis consequuntur fuga assumenda magni amet cum Iusto enim fugiat provident aliquam nobis laborum aperiam similique aspernatur obcaecati sunt asperiores quidem totam numquam at hic fuga temporibus Distinctio commodi nostrum laudantium accusantium quam.
    </p>
    <p
      className="paragraph2"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas suscipit deserunt commodi iusto quasi voluptatem incidunt modi perspiciatis ipsam dolorum at ut pariatur sunt veniam saepe, eveniet quis consequuntur fuga assumenda magni amet cum Iusto enim fugiat provident aliquam nobis laborum aperiam similique aspernatur obcaecati sunt.
    </p>
  </main>,
  <footer>
    <span
      className="footer-text"
    >
      Created by Kashif Saleem
    </span>
    <a
      className="github"
      href="https://github.com/Kashif-Saleem-Ghuman"
      rel="noreferrer"
      target="_blank"
    >
      Github
    </a>
  </footer>,
]
`;
